### ISSUE TITLE
Ability to convert mobile numbers to international format when talking to SMS gateway

### ISSUE THREAD

**Issue by tbar0970** [tbar0970/jethro-pmm#107]
In the conf file you can configure templates for talking to SMS gateways.
Some gateways require the numbers to be in international format (eg +61417123456 instead of 0417123456).
It would be valuable to have some way of making this possible in the configuration.
Perhaps some sort of template for number conversion
```
    define('MOBILE_LOCAL_PREFIX', '0') // remove this from the front of the number when converting to international
    define('MOBILE_INTERNATIONAL_PREFIX', '+61') // add add this to the front of the number when converting to international
````

**Comment by vanoudt** [comment]
Some work towards this in my pull request #120

**Comment by tbar0970** [comment]
This is resolved in aa0f000860008e56bb024c09da28e4f4c100faae based on @vanoudt 's pull request
